src/js/App/Header/AppFilter.js

```diff
@@ -66,43 +67,53 @@ const AppFilterDropdown = ({ isLoaded, setIsOpen, isOpen, filterValue, setFilter
     isOpen={isOpen}
     ouiaId="App Filter"
   >
-    <div className="content">
-      {isLoaded ? (
-        <React.Fragment>
-          <Flex className="search">
-            <SearchInput
-              data-ouia-component-id="app-filter-search"
-              placeholder="Find application or service"
-              value={filterValue}
-              onChange={(val) => setFilterValue(val)}
-              onClear={() => setFilterValue('')}
-            />
-          </Flex>
-          {filteredApps?.length > 0 ? (
-            <div className="gallery">
-              {filteredApps.map((app) => (
-                <App key={app.id} {...app} />
-              ))}
+    {ReactDOM.createPortal(
+      <>
+        <div className="pf-c-dropdown ins-c-page__app-filter-dropdown-menu">
+          <div className="pf-c-dropdown__menu">
```

Comment:
Why not using the direct `Dropdown` component?

Reply:
I wanted to create a really lightweight cloned structure to append to the document body